src/encoded/static/img/bodyMap/Deselected-mouse.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+
+const SVG = ({ handleClick, handleHighlight, BodyList }) => (
+    /* eslint-disable */
```

Comment:
I got rid of this entirely. I think I had it because I needed to do a find-and-replace on the human diagram and Atom kept crashing but I got Atom to cooperate so this is much better.